tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -217,6 +217,7 @@ protected override bool Skip (Type type, string protocolName)
 				// iOS 11.3
 				case "PKSuicaPassProperties": // Conformance not in headers
 				case "PKTransitPassProperties": // Conformance not in headers
+				case "ARReferenceImage": // Conformance removed from headers in Xcode 9.3 Beta 4
```

Comment:
comment is not quite true - it's more like the above `// Conformance not in headers`
not worth another build